astrowidgets/__init__.py

```diff
@@ -12,10 +12,10 @@
 
 __minimum_python_version__ = "3.5"
 
-class UnsupportedPythonError(Exception):
-    pass
-
 if sys.version_info < tuple((int(val) for val in __minimum_python_version__.split('.'))):
+    class UnsupportedPythonError(Exception):
```

Comment:
Note: Need to bury the exception class here to avoid `automodapi` to pick it up.